src/main/scala/io/viash/platforms/DockerPlatform.scala

```diff
@@ -156,7 +156,16 @@ case class DockerPlatform(
 
   @description("Additional requirements specific for running unit tests.")
   @since("Viash 0.5.13")
-  test_setup: List[Requirements] = Nil
+  test_setup: List[Requirements] = Nil,
+
+  @description("Override the entrypoint of the base container. Default set `ENTRYPOINT []`.")
+  @since("Viash 0.7.4")
+  entrypoint: Option[String] = Some("[]"),
```

Comment:
Would this work?

```suggestion
  entrypoint: Option[List[String]] = Some(Nil),
```

Does this allow to differentiate between `None` (no entrypoint should be added) and `Some(Nil)` (entrypoint should be `[]`)

Reply:
I changed it to `entrypoint: Option[Either[String, List[String]]] = Some(Right(Nil)),` as it allows [the full described functionality](https://docs.docker.com/engine/reference/builder/#entrypoint) of either single string or an array of strings.
The single string is a non-preferred option but the actual additional code to support this is quite minimal.
The biggest issue was the distinction between a null value and an empty string.